docs/navigation-configuration.md

```diff
@@ -217,6 +217,9 @@ You can configure the way Luigi tackles routing in your application in the **Rou
 - **useHashRouting** defines either hash-based (`url.com/#/yourpath`) or path-based (`url.com/yourpath`) routing.
 - **nodeParamPrefix** sets the prefix character when using the `LuigiClient.linkManager().withParam()` function, which provides a way to simply attach query parameters to the view URL for activities such as sorting and filtering. The URL contains the parameters to allow deep linking. If you want to use a different character prefix, define yours here. The default character is `~`.
 - **skipRoutingForUrlPatterns** defines regex patterns to be skipped by the router when listening for path changes. This parameter is used for excluding **redirect_uri** parameters. Default patterns are `[/access_token=/, '/id_token=/]`.
+- **pageNotFoundHandler** is a function defining custom behaviour when the 404 (page not found) error occured. By default it's handled by Luigi. Leave its body empty if you have an external 404 handling. It takes two parameters: 
+  - **wrongPath**(string): the path that user tried to navigate to
+  - **wasAnyPathFitted**(bool): is true if Luigi managed to fit a valid path which means _wrongPath_ was only "partly wrong". Otherwise it is false.
```

Comment:
```suggestion
  - **wasAnyPathFitted**(bool): it is true if Luigi managed to fit a valid path which means **wrongPath** was only partially wrong. Otherwise it is false.
```